src/net_processing.cpp

```diff
@@ -1063,12 +1067,63 @@ bool static AlreadyHave(const CInv& inv) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
 static void RelayTransaction(const CTransaction& tx, CConnman* connman)
 {
     CInv inv(MSG_TX, tx.GetHash());
-    connman->ForEachNode([&inv](CNode* pnode)
-    {
+    connman->ForEachNode([&inv](CNode* pnode) {
         pnode->PushInventory(inv);
+        pnode->m_cache_dandelion.erase(inv.hash);
     });
 }
 
+static void FluffTransaction(CTxMemPool& tx_pool, const CNode* from, const CTransactionRef& ptx, CConnman* connman)
```

Comment:
I was wandering... FluffTransaction is logically equivalent to receiving a transaction, right?
Then shouldn't we also execute all [this code](https://github.com/bitcoin/bitcoin/blob/423cb37658b81b2cb8475b6fdb08b3122fee51a4/src/net_processing.cpp#L2190) on fluff?